### PR TITLE
[ADHOC] Managed Budget component page

### PR DESCRIPTION
**Pull request by mmackz**
~Draft of~ Managed budget component page explaining each method and showing an example.

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.text-yellow {
```
Comment (mmackz) [inline styles/styles.css:1]: Tried using tailwind classes for color and margin, but they dont seem to work. 

They do work for font and text size however 🤷‍♂️

File: mint.json [hunk 2]
```diff
@@ -215,6 +215,14 @@
         "v2/boost-sdk/examples/zora-mint",
         "v2/boost-sdk/examples/agora-vote"
       ]
+    },
+    {
```
Comment (sammccord) [inline mint.json]: can we use a different top level group per type of component?
like 
```json
{
"group": "Budgets",
"version": "v2",
"pages": [
  { "group": "Managed Budget", "pages": [ "v2/boost-sdk/budgets/managed/overview" ] }
 ]
}
```
?

File: v2/boost-sdk/components/ManagedBudget/managedBudget.mdx [hunk 3]
```diff
@@ -0,0 +1,197 @@
+---
+title: "ManagedBudget"
+---
+import Config from '/snippets/config.mdx';
```
Comment (sammccord) [inline v2/boost-sdk/components/ManagedBudget/managedBudget.mdx]: can we also include the sdk disclaimer snippet to all of these pages?

File: v2/boost-sdk/components/ManagedBudget/managedBudget.mdx [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+  { chainId: 8453 }
+)
+```
+</CodeGroup>
+
+<Tip>
+  [Learn more about targeting specific chains](/v2/boost-sdk/quick-start#targeting-specific-chains)
+</Tip>
+
+### Write Methods
```
Comment (sammccord) [inline v2/boost-sdk/components/ManagedBudget/managedBudget.mdx]: for consistency, let's merge write and read

### API

#### allocate (same case as in source code for public methods)

Comment (mmackz) [reply v2/boost-sdk/components/ManagedBudget/managedBudget.mdx]: Was an old draft. Will update it to be more inline with the other pages.

File: v2/boost-sdk/components/ManagedBudget/managedBudget.mdx [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+// or if you want a budget from a specific chain
+const budgetOnBase = core.ManagedBudget(
+  "0xc55F719709bDad022B320E76f9DfF7e6F5680767", 
+  { chainId: 8453 }
+)
+```
+</CodeGroup>
+
+<Tip>
+  [Learn more about targeting specific chains](/v2/boost-sdk/quick-start#targeting-specific-chains)
```
Comment (sammccord) [inline v2/boost-sdk/components/ManagedBudget/managedBudget.mdx]: 👍🏼

File: v2/boost-sdk/components/ManagedBudget/managedBudget.mdx [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+  target: account.address,
+});
+```
+</CodeGroup>
+
+> [Types](https://boost-protocol.vercel.app/classes/ManagedBudget.html#allocate)
+
+<ReturnValue value="Promise<boolean>" description="true if the request was successful" />
+
+<Note>
```
Comment (sammccord) [inline v2/boost-sdk/components/ManagedBudget/managedBudget.mdx]: 👍🏼

**Comment by sammccord** [comment]
this is looking really good so far

File: v2/boost-sdk/budgets/managed/authorization.mdx [hunk 7]
```diff
@@ -0,0 +1,276 @@
...
+
+import SDKDisclaimer from '/snippets/sdk-disclaimer.mdx';
+import ReadParams from '/snippets/read-params.mdx';
+import WriteParams from '/snippets/write-params.mdx';
+
+<SDKDisclaimer />
+
+A budget has two types of roles that can be granted: `ADMIN` and `MANAGER`. These roles can be accessed through the `Roles` enum:
+```ts
+import { Roles } from "boost-sdk"
```
Comment (sammccord) [inline v2/boost-sdk/budgets/managed/authorization.mdx]:
```suggestion
import { Roles } from "@boostxyz/sdk"
```

**Review by sammccord — APPROVED** [review]

**Pull request merged**